edgelet/edgelet-core/src/settings.rs

```diff
@@ -396,10 +396,33 @@ impl External {
     }
 }
 
+#[derive(Clone, Debug, serde_derive::Deserialize, serde_derive::Serialize)]
+#[serde(rename_all = "lowercase")]
+pub struct Provisioning {
+    #[serde(flatten)]
+    provisioning: ProvisioningType,
+
+    #[serde(
+        rename = "dynamic_reprovisioning",
+        skip_serializing_if = "Option::is_none"
+    )]
+    dynamic_reprovisioning: Option<bool>,
```

Comment:
Since `None` is treated the same as `Some(false)`, you could do:

```rust
#[serde(default)]
dynamic_reprovisioning: bool,
```

instead.

Reply:
Sweet, thanks for the suggestion!